deepspeed/moe/utils.py

```diff
@@ -59,8 +59,9 @@ def split_params_grads_into_shared_and_expert_params(
     return shared_grads, expert_grads
 
 
-def split_params_into_different_moe_groups_for_optimizer(
-        param_groups: Tuple[Dict]) -> Tuple[Dict]:
+def split_params_into_different_moe_groups_for_optimizer(param_groups: Tuple[Dict],
```

Comment:
@tjruwase, please let me know how would we want to offer the user a way to set the max_group_size

Reply:
@awan-10

Reply:
@awan-10 Perhaps the moe section can also contain a flag to toggle expert slicing too?